x-pack/test/send_search_to_background_integration/tests/apps/dashboard/async_search/sessions_in_space.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+import { getSearchSessionIdByPanelProvider } from './get_search_session_id_by_panel';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+  const esArchiver = getService('esArchiver');
+  const security = getService('security');
+  const PageObjects = getPageObjects([
+    'common',
+    'header',
+    'dashboard',
+    'visChart',
+    'security',
+    'timePicker',
+  ]);
+  const getSearchSessionIdByPanel = getSearchSessionIdByPanelProvider(getService);
+  const browser = getService('browser');
+  const sendToBackground = getService('sendToBackground');
+
+  describe('dashboard in space', () => {
+    describe('Send to background in space', () => {
+      before(async () => {
+        await esArchiver.load('dashboard/session_in_space');
+
+        await security.role.create('data_analyst', {
+          elasticsearch: {
+            indices: [{ names: ['logstash-*'], privileges: ['all'] }],
+          },
+          kibana: [
+            {
+              base: ['all'],
+              spaces: ['another-space'],
+            },
+          ],
+        });
+
+        await security.user.create('analyst', {
+          password: 'analyst-password',
+          roles: ['data_analyst'],
+          full_name: 'test user',
+        });
+
+        await PageObjects.security.forceLogout();
+
+        await PageObjects.security.login('analyst', 'analyst-password', {
+          expectSpaceSelector: false,
+        });
+      });
+
+      after(async () => {
+        await esArchiver.unload('dashboard/session_in_space');
+        await PageObjects.security.forceLogout();
+      });
+
+      it('Saves and restores a session', async () => {
```

Comment:
🥇 🎉 👏 🙌

Reply:
@lukasolson we're not doing visualize ATM

Reply:
Ah I see we're not doing visualize. Ignore my earlier comment then.